_sass/kroxylicious.scss

```diff
@@ -117,4 +117,16 @@ $navbar-dark-disabled-color: rgba($white, 0.45);
 
 div.highlighter-rouge {
   padding: 0 10px;
-}
\ No newline at end of file
+}
+
+// Hero Styles
+
+.krx-hero {
+  background: rgb(73,160,120);
+  background: linear-gradient(111deg, rgba(73,160,120,1) 0%, rgba(44,96,72,1) 30%, rgba(22,48,36,1) 100%);
+
+  h1, p {
+  color: $white !important;
```

Comment:
I don't have a problem with doing it this way, but worth noting the property `data-bs-theme="dark"` would achieve the same effect on all child text.

Reply:
Seems I was wrong because the `<p>` text in the hero isn't inheriting the colour for some reason. I'm happy to leave this as it is.